lib/linalg/svd/RandomizedSVD.h

```diff
@@ -26,9 +26,9 @@ namespace CAROM {
 
 /**
  * Class RandomizedSVD implements the Randomized SVD algorithm introduced in
- *    "Finding Structure with Randomness: Probabilistic Algorithms for
- *    Constructing Approximate Matrix Decompositions" by N. Halko, P. G.
- *    Martinsson, and J. A. Tropp
+ *    Halko, Nathan, Per-Gunnar Martinsson, and Joel A. Tropp.
+ *    "Finding structure with randomness: Probabilistic algorithms for constructing approximate matrix decompositions."
+ *    SIAM review 53.2 (2011): 217-288.
```

Comment:
Not a very important comment: for consistency in the style of the author names, the first author should be "Nathan Halko". Maybe "SIAM Review Vol. 53, Iss. 2 (2011): 217-288" looks better.

Reply:
Reflected.